user-interface/src/data-verification/consolidation/ConsolidationOrderAccordion.test.tsx

```diff
@@ -102,11 +58,18 @@ describe('ConsolidationOrderAccordion tests', () => {
       'consolidations-enabled': true,
     };
     vitest.spyOn(FeatureFlagHook, 'default').mockReturnValue(mockFeatureFlags);
-    setupApiGetMock();
+    vitest
```

Comment:
We should probably replace `vitest.spyOn` with `vi.spyOn` across all tests. But probably in a subsequent PR.